swupd/create_manifests_test.go

```diff
@@ -567,3 +567,61 @@ func mustParseAllManifests(t *testing.T, version uint32, outputDir string) map[s
 
 	return result
 }
+
+// Imported from swupd-server/test/functional/subtract-delete.
+func TestSubtractDelete(t *testing.T) {
+	ts := newTestSwupd(t, "subtract-delete-")
+	defer ts.cleanup()
+
+	ts.Bundles = []string{"os-core", "test-bundle"}
+	ts.Format = 3
```

Comment:
You don't seem to change this value again. Why 3? I recall it was 3 in the old server because Clear Linux was on format 3 when the tests were originally written, but I don't think it is relevant here.

Reply:
Yes, I was just keeping the test doing as much as possible the same as before -- I guessed it was an historical reason or something. Having some tests using different formats is a win, but I think we have those around already.

I'll remove format 3 from tests that don't really depend on that.